src/components/inputs/select-input/select-input.js

```diff
@@ -77,7 +107,7 @@ export class SelectInput extends React.Component {
           )
         : optionsWithoutGroups.find(
             option => has(option, 'value') && option.value === props.value
-          ),
+          ) || [],
```

Comment:
This should be `null`, as we're in the single-value mode here. 

@islam3zzat was working on this as well. More information in https://github.com/commercetools/ui-kit/pull/19.